srcpkgs/kmscon/template

```diff
@@ -1,19 +1,19 @@
 # Template file for 'kmscon'
 pkgname=kmscon
-version=9.0.0
-revision=1
+version=9.2.0
+revision=2
 build_style=meson
-configure_args="-Dwerror=false -Dtests=false"
+configure_args="-Dwerror=false -Dtests=false -Ddocs=enabled"
 hostmakedepends="pkg-config docbook-xsl xkeyboard-config libxslt"
 makedepends="MesaLib-devel pango-devel libtsm-devel libxkbcommon-devel
- eudev-libudev-devel"
+ eudev-libudev-devel libdrm-devel libgbm-devel"
 depends="xkeyboard-config"
 short_desc="Terminal emulator based on Kernel Mode Setting (KMS)"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Huang FTS427 <FTS427@outlook.com>"
```

Comment:
Please remove the adoption. We would like to see maintainers establish a history of good stewardship before putting their names on packages.